pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/bestpractices/xml/ApexUnitTestClassShouldHaveAsserts.xml

```diff
@@ -115,4 +115,18 @@ public class Foo {
 }
         ]]></code>
     </test-code>
+
+    <test-code>
+      <description>#4096 [apex] api 56.0 ApexAssertionsShouldIncludeMessage and new apex class : Assert</description>
```

Comment:
```suggestion
      <description>[apex] ApexUnitTestClassShouldHaveAssertsRule: Support new Assert class (Apex v56.0) #4097</description>
```

The issue #4096 is actually about the rule ApexAssertionsShouldIncludeMessage, but this PR changes the rule ApexUnitTestClassShouldHaveAsserts.

The rule ApexAssertionsShouldIncludeMessage probably also needs to be adjusted.